gateway/validate.py

```diff
@@ -0,0 +1,55 @@
+import subprocess
+from abc import ABC, abstractmethod
+from typing import List, Optional
+
+class OCFLFixityValidator(ABC):
+    @abstractmethod
```

Comment:
Just reiterating the question here of whether validate methods should be within the `OCFLRepositoryGateway` or not. I can see arguments before and against. If you did put it in the gateway, you'd be keeping all the `rocfl` stuff in one place, but then you're assuming other implementations we might explore also have Fixity checking (which they should but might not). Not sure of the answer, but thought I'd flag the question.

Reply:
I think it's OK to have them be separate. We can reevaluate as needed.